src/flb_parser.c

```diff
@@ -1008,6 +1025,11 @@ int flb_parser_tzone_offset(const char *str, int len, int *tmdiff)
         return 0;
     }
 
+    /* Check system timezones */
+    if (*p == 'S') {
```

Comment:
Probably worth doing a `strncmp` here instead of just checking the first character.